api/stream-payment-order/openapi.yaml

```diff
@@ -0,0 +1,1054 @@
+openapi: 3.0.1
+info:
+  title: Payment Order API
+  version: 1.0.0
+  x-logo:
+    url: "http://www.backbase.com/wp-content/uploads/2017/04/backbase-logo-png.png"
+    backgroundColor: "#FFFFFF"
+    altText: "Backbase logo"
+servers:
+  - url: http://stream-product-catalog-http.proto.backbasecloud.com/
+    description: The server
+paths:
+  ###############################################
+  ## Product Catalog
+  ###############################################
+  /payment-order:
+    get:
+      summary: Get Payment Order
+      operationId: getPaymentOrder
+      tags:
+        - payment-order
+      description: Assemble Payment Order
+      responses:
+        200:
+          description: The created payment type
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PaymentOrder'
+        default:
+          description: The
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
+      summary: Setup a Product Catalog
+      tags:
```

Comment:
Why product catalog here ? And in other places of this spec

Reply:
I removed this file until we build events. There is another ticket for that.